Applications/TransferLearning/Draw_Classification/jni/main.cpp

```diff
@@ -36,20 +34,9 @@
 #include <time.h>
 #include <unistd.h>
 
-#if defined(__TIZEN__)
 #include <nnstreamer-single.h>
 #include <nnstreamer.h>
 #include <nntrainer_internal.h>
-#else
-#include "tensorflow/contrib/lite/interpreter.h"
```

Comment:
Don't need to delete this example. You could make another application for this I guess.

Reply:
This application change will have to revert as the nnstreamer_layer is not supported on android.
I will make a tflite_layer for backend and update example with that.